fluentd/configs.d/input-post-forward-mux.conf

```diff
@@ -38,6 +38,14 @@
     @label @INGRESS
   </match>
 
+  # if these have already been processed and formatted by an openshift fluentd, skip
```

Comment:
Is this a conditional?  Is there a test here to see if records tagged with `journal.container**` or `journal.system` are already processed?  Or are should this comment read something like, "`All journal.container** and journal.system tagged records have already been processed by an openshift fluentd, so skip any additional processing by retagging them as mux.ops.`"